code/modules/mob/living/simple_animal/hostile/bat.dm

```diff
@@ -78,7 +78,7 @@
 	..()
 	if(emergency_shuttle.location == 1)
 		if(!enroute && !target)	//The shuttle docked, all monsters rush for the escape hallway
-			if(!shuttletarget || (get_dist(src, shuttletarget) >= 2))
+			if(!shuttletarget && escape_list.len && (get_dist(src, shuttletarget) <= 2))
```

Comment:
It is required, because get_dist will fail on a null shuttletarget.
So it should be
`if(escape_list.len && (!shuttletarget || (get_dist(src, shuttletarget) >= 2)))`